docker-compose.yml

```diff
@@ -63,3 +63,10 @@ services:
       - "10000:10000"  # so redirect urls for images work
     logging:
       driver: none
+
+  # this image depends on a custom prebuild script so we can't define the build context here
+  predict_api:
```

Comment:
docker-compose files support composition, so you can start it like "docker-compose up -f docker-compose.yml -f predict.yml" so frontend developers don't wait for all the related stuff to start

Reply:
thanks, I pushed a commit to split it out to a separate file.